<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMDDataBlockDialog</class>
 <widget class="QDialog" name="CMDDataBlockDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>342</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>描述</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="descriptionEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rateEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">1000</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>扫描周期(ms)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="startAddressEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>开始地址</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="enableCheckBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>启用</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="endAddressEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>结束地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>寄存器类型</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QComboBox" name="registerComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>X</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Y</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>D</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>W</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>L</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>V</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>S</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SW</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>R</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Z</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CN</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
